sdk/python/README.md

```diff
@@ -0,0 +1,43 @@
+# kubeflow
+Python SDK for MPI-Operator
+
+This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
+
+- API version: v1
+- Package version: 0.1
+- Build package: Kubeflow-MPIJob
+
+## Requirements.
+
+Python 3.6+
+
+## Installation & Usage
+### Setuptools
```

Comment:
We will not push this SDK to PyPi ?
cc @terrytangyuan @gaocegege

Reply:
I am not sure if it's worth to maintain it in PyPI. It's easy to get out-of-date. Users can always install via `pip git+xxxx` anyways.

Reply:
I think it would be great to add this install command to the README.

Reply:
sure. I updated the README and add `pip install git+xxxx` in the installation section.